### DISCUSSION TITLE
does proxqp optimize the  case when objective matrix(H) is dense but inequality constraint matrix(C) is very sparse,

### DISCUSSION THREAD

**Discussion by pengfzhou** [Simple-Robotics/proxsuite#217] [Q&A] [answered]

**Comment by pengfzhou** [comment]
also can  proxqp deal with simple box bound efficiently if a problem only have box constraint

**Comment by jcarpent** [comment, marked as answer]
If H is dense, it is thus better to use the dense backend. The sparse backend is still under development.
What is the size of your problems? Could you share an example?

> also can proxqp deal with simple box bound efficiently if a problem only have box constraint

We can handle bound constraints as generic inequality constraints. Shortly, we will add explicit support for bound constraints for efficiency reasons.

**Comment by jcarpent** [comment]
Related to https://github.com/Simple-Robotics/proxsuite/issues/47
